Tests/StreamChatTests/Controllers/ChannelListController/ChannelListController_Tests.swift

```diff
@@ -1095,6 +1092,268 @@ final class ChannelListController_Tests: XCTestCase {
         XCTAssert(client.activeChannelListControllers.count == 1)
         XCTAssert(client.activeChannelListControllers.allObjects.first === controller)
     }
+
+    // MARK: Predicates
+
+    private func assertFilterPredicate(
```

Comment:
Because before, we didn't have any local filtering, so we were accepting whatever the server was sending us (in this case the mocked data). 
Now, that the local filtering is in place for the data to actually be retrieved they need to match the filter.

In those cases here, the used filter is checking that the user is a member of those channels. So, my changes are making sure that every channel that is being tested, has the required user as a member

Reply:
This is usually my approach, but on this specific file I noticed that `private` functions have been recently added close to their test cases and that's why i have this one here